sdk/eventhubs/azure-messaging-eventhubs-stress/stress-test-resources.bicep

```diff
@@ -1,13 +1,13 @@
 @description('The base resource name.')
-param baseName string = resourceGroup().name
+param baseResourceName string = resourceGroup().name
 
 @description('The location of the resources. By default, this is the same as the resource group.')
 param location string = resourceGroup().location
 
-var subBaseName = '${substring(baseName, 0, min(length(baseName), 12))}-${substring(guid(baseName), 0, 4)}'
-var eventHubsNamespaceName = '${subBaseName}-ehns'
+var subBaseName = '${substring(baseResourceName, 0, min(length(baseResourceName), 22))}-${substring(guid(baseResourceName), 0, 5)}'
```

Comment:
The `baseName` parameter we pass now is a truncated hash of the resource group name that is valid for all azure resources. So you can get rid of this if you want, unless you want to keep some of the human readable aspect to it.

Reply:
yes, I did this changes because when I I found our namespace started with the hashcode, which makes me suspect if it was the resource I had created for a while:laughing:. 
I still want to keep using the resource group name as base name since it is more readable. I am thinking that if we are running several tests, and have multiple windows open on browser, a more readable namespace could somehow help us to swith between them.